ola.spec.in

```diff
@@ -92,10 +91,7 @@ cp debian/ola.udev %buildroot/%{_udevrulesdir}
 %{_bindir}/rdmpro_sniffer
 %{_bindir}/usbpro_firmware
 %{_datadir}/olad
-%{_datadir}/olad/**
-%{_datadir}/ola
```

Comment:
Would you not be better off dropping ola/pids in case we add more stuff in this folder in future?